src/result_collection.jl

```diff
@@ -90,7 +90,7 @@ function collect_results!(filename, folder;
     end
 
     n = 0 # new entries added
-    existing_files = get(df, :path, ())
```

Comment:
I assume that `get(df, :path, ())` was deprecated? Too bad, the alternative `:path in names(df) ? df[:,:path] : ()` is unreasonably verbose :(

Reply:
Sadly it was.